appserver/website-bootstrap/network.js

```diff
@@ -1,3 +1,218 @@
+const network_svg_id= "network-svg";
+
 $(document).ready(function(){
 
-});
\ No newline at end of file
+    document.getElementById('update-graph-btn').addEventListener('click', function () {
+        // example URL /retweeters?username=Drake&count=20
+        const svgPresent = !! document.getElementById(network_svg_id);
+        if (svgPresent) { d3.select("#" + network_svg_id).remove(); }
+        let retweetNetworkURL = 'http://18.214.197.203:5000/retweeters?username=';
+        let uname = document.getElementById('search-network').value;
+        let retweets = document.getElementById('drop-list-network').value;
```

Comment:
this is referring to the number of retweeters who should be displayed, right? It's a little confusing imo that it's called retweets instead of retweeters or retweeter_count

Reply:
fixed --> retweeter_count